src/applications/personalization/profile360/containers/VAProfileApp.jsx

```diff
@@ -21,22 +21,30 @@ import RequiredLoginView from '../../../../platform/user/authorization/component
 import ProfileView from '../components/ProfileView';
 
 class VAProfileApp extends React.Component {
-  componentDidMount() {
-    this.props.fetchTransactions();
+  componentDidUpdate(oldProps) {
+    const justLoggedIn = !oldProps.user.login.currentlyLoggedIn && this.props.user.login.currentlyLoggedIn;
+    if (justLoggedIn) {
+      if (this.props.isVet360AvailableForUser) {
+        this.props.fetchTransactions();
+      } else {
+        // this.props.initializeUserToVet360()
+      }
+    }
   }
+
```

Comment:
Rather than check for a `null` value for the `profile.vet360`, we use the selector `isVet360AvailableForUser` that checks for `vet360` in the `profile.services` list.